django-stubs/contrib/admin/options.pyi

```diff
@@ -150,33 +150,33 @@ _ActionCallable: TypeAlias = Callable[[_ModelAdmin, HttpRequest, QuerySet[_Model
 class ModelAdmin(BaseModelAdmin[_ModelT]):
     list_display: _DisplayT[_ModelT]
```

Comment:
Please, add a comment: why do we left these two without a `ClassVar`, so future users won't be confused.

Reply:
Added in 3e2fdc8.